lenovo/thinkpad/general.nix

```diff
@@ -12,11 +12,11 @@ with lib;
   services.tlp.enable = true;
 
   services.xserver = {
-    synaptics.enable = false;
-    libinput.enable = true;
+    synaptics.enable = mkDefault false;
+    libinput.enable = mkDefault true;
```

Comment:
Those two should be in one `mkDefault`, so that either both get set or neither, I’d say.

Reply:
done